OCPBUGS#43947: Updated the OPENSHIFT_HA_NETWORK_INTERFACE description (#89118)
openshift/openshift-docs: modules/nw-ipfailover-environment-variables.adoc

|`OPENSHIFT_HA_NETWORK_INTERFACE`
|
|The interface name that IP failover uses to send Virtual Router Redundancy Protocol (VRRP) traffic. The default value is `eth0`. 

> If your cluster uses the OVN-Kubernetes network plugin, assign `br-ex` as the value for `OPENSHIFT_HA_NETWORK_INTERFACE` so that no packet loss occurs. For a cluster that uses the OVN-Kubernetes network plugin, all listening interfaces do not serve VRRP but instead expect inbound traffic over a `br-ex` bridge.

Reviewer: ```suggestion
If your cluster uses the OVN-Kubernetes network plugin, set this value to `br-ex` to avoid packet loss. For a cluster that uses the OVN-Kubernetes network plugin, all listening interfaces do not serve VRRP but instead expect inbound traffic over a `br-ex` bridge.
```
